README.md

```diff
@@ -3,17 +3,17 @@
 This is the service api for persisting application related information to the back end database and
 may well be used to fire requests to other services.
 
-* Ruby version
+* Ruby version 
```

Comment:
As discussed face to face, there are still whitespace changes in this branch that should not be there.